docs/reference/changelog.md

```diff
@@ -38,6 +39,7 @@
   * milestoneType
   * milestoneStatus
   * milestoneCode
+* [#427](https://github.com/open-contracting/infrastructure/pull/427) - add country codelist.
```

Comment:
Would be clearer to format as:

```md
* add codelists:
  * [#369](https://github.com/open-contracting/infrastructure/pull/369) - classificationScheme
  * [#427](https://github.com/open-contracting/infrastructure/pull/427) - country
  * [#426](https://github.com/open-contracting/infrastructure/pull/426) - milestoneType
  * [#426](https://github.com/open-contracting/infrastructure/pull/426) - milestoneStatus
  * [#426](https://github.com/open-contracting/infrastructure/pull/426) - milestoneCode
```

Reply:
Done in 29cb3e0cce565b00e0e4e2ab82178bc68a2bbebe